tasks/tickets.rake

```
@@ -43,7 +30,7 @@ def validate_vars(jira, vars)
   jira.user? vars[:tester]
 end
```

Same assumption as earlier, that the `jira` "question mark" instance methods actually `raise` instead of returning a boolean?